benchmark/run.js

```diff
@@ -10,16 +10,26 @@ const cli = CLI(`usage: ./node run.js [options] [--] <category> ...
 
   --filter pattern          string to filter benchmark scripts
   --set    variable=value   set benchmark variable (can be repeated)
+  --format simple/csv       optional value that specifies the output format
 `, {
   arrayArgs: ['set']
 });
 const benchmarks = cli.benchmarks();
+const validFormats = ['csv', 'simple'];
 
 if (benchmarks.length === 0) {
   console.error('no benchmarks found');
   process.exit(1);
 }
 
+if (cli.optional.format) {
```

Comment:
Use `console.error` and the `return` is not required (like the `benchmark.length` test above). Exit codes can only be non negative (`code >= 0`). You can also shorten it a bit.

```js
const format = cli.optional.format || 'simple';
if (!['csv', 'simple'].includes(format)) {
  console.error('Invalid format detected');
  process.exit(1);
}
```